python/cuml/common/array.py

```diff
@@ -168,6 +169,12 @@ def __setitem__(self, slice, value):
     def __len__(self):
         return self.shape[0]
 
+    def _operator_overload(self, other, fn):
+        return CumlArray(fn(self.to_output('cupy'), other))
+
+    def __add__(self, other):
```

Comment:
Just for potential future regressions, it would be nice to see a super simple pytest that adds two cuml arrays.